faith_presentation/src/commonMain/kotlin/net/thechance/mena/faith/presentation/feature/mosque/pickLocationMap/PickLocationScreenInteractionListener.kt

```diff
@@ -0,0 +1,12 @@
+package net.thechance.mena.faith.presentation.feature.mosque.pickLocationMap
+
+
+internal interface PickLocationScreenInteractionListener {
+    fun onClickMap(coordinates: CoordinatesUiState)
+    fun onMoveCamera(coordinates: CoordinatesUiState)
+    fun onClickGps()
+    fun onClickConfirm()
+    fun onClickBack()
```

Comment:
```suggestion
    fun onMapClick(coordinates: CoordinatesUiState)
    fun onMoveCamera(coordinates: CoordinatesUiState)
    fun onGpsClick()
    fun onConfirmClick()
    fun onBackClick()
```

just for consistency